drivers/slipdev/include/slipdev_internal.h

```diff
@@ -43,11 +43,51 @@ extern "C" {
 /**
  * @brief   Marker byte for beginning of stdio
  * @see     taken from diagnostic transfer from
- *          [SLIPMUX](https://tools.ietf.org/html/draft-bormann-t2trg-slipmux-02#section-4)
+ *          [SLIPMUX](https://tools.ietf.org/html/draft-bormann-t2trg-slipmux-03#section-4)
  */
 #define SLIPDEV_STDIO_START (0x0aU)
+
+/**
+ * @brief   Marker byte for beginning of configuration/coap
+ * @see     taken from configuration from
+ *          [SLIPMUX](https://tools.ietf.org/html/draft-bormann-t2trg-slipmux-03#section-5)
+ */
+#define SLIPDEV_CONFIG_START (0xa9U)
 /** @} */
 
+
+/* Special init
+ * the result of normal fcs init combined with slipmux config start (0xa9)
+ */
+#define SPECIAL_INIT_FCS (0x374cU)
+#define INIT_FCS (0xffffU)
+#define GOOD_FCS (0xf0b8U)
+
+static const uint16_t FCS_LOOKUP[256] = {
```

Comment:
Not sure why Doxygen is complaining here. Maybe it helps to already put them in a Doxygen group?